docs/versioned_docs/version-2.11.0/deprecated/ugcstor.md

```diff
@@ -0,0 +1,2496 @@
+---
+id: ugcstor
```

Comment:
The ugcstor doesnot make any sense here, ug => user guides

PTAL the suggestion or any relevant id
Please change the filename also
```suggestion
id: spc-based-cstor
```

Reply:
Please change the file name as well